backend/src/App.js

```diff
@@ -26,11 +37,29 @@ export const app = async () => {
   restApiServer.use(express.json());
   restApiServer.use(express.urlencoded({ extended: true }));
   restApiServer.use(morgan('combined'));
+  restApiServer.use(flash());
+  restApiServer.use(
+    session({
+      secret: 'dwadwadagwafa',
+      resave: false,
+      saveUninitialized: false,
+    }),
+  );
+  restApiServer.use(passport.initialize());
+  restApiServer.use(passport.session());
+  restApiServer.use(methodOverride('_method'));
   restApiServer.use('/rest-api', userRegistrationDetailsController.router);
 
   return restApiServer;
 };
 
+const initializePassport = initialize;
+initializePassport(
+  passport,
+  email => mongoDocumentToDomain.find(user => user.email === email),
+  id => mongoDocumentToDomain.find(user => user.userId === id)
+)
+
```

Comment:
Coś tutaj sięgamy z pominięciem service :) Czy aby na pewno dobra droga?

Reply:
Może tym powinien zajmować się inny moduł, albo inna metoda w module UserRegistration?

Reply:
Albo jeśli to bedzie dostępne dla wszystkich modułów to wrzucić w `src/utils` albo jeśli zawsze w każdym module to w `src/common`